test/Sema/compound_references.swift

```diff
@@ -0,0 +1,4 @@
+// RUN: %target-typecheck-verify-swift
+
+func foo(x: Int) {} // expected-note * {{did you mean 'foo(x:)'?}}
+let f = foo(_:) // expected-error {{use of unresolved identifier 'foo'}}
```

Comment:
Please remove this file and add these tests to `Sema/typo_correction.swift` (sans * on the `expected-note`).